openreview/venue_request/process/revisionProcess.py

```diff
@@ -132,6 +184,12 @@ def process(client, note, invitation):
                     submission_readers=[openreview.SubmissionStage.Readers.SENIOR_AREA_CHAIRS_ASSIGNED, openreview.SubmissionStage.Readers.AREA_CHAIRS_ASSIGNED, openreview.SubmissionStage.Readers.REVIEWERS_ASSIGNED]
 
             conference.post_decision_stage(reveal_all_authors,reveal_authors_accepted,decision_heading_map=forum_note.content.get('home_page_tab_names'), submission_readers=submission_readers)
+            if forum_note.content['send_decision_notifications'] == 'Yes, send an email notification to the authors':
+                decision_options = forum_note.content.get('decision_options')
+                decision_options = [s.translate(str.maketrans('', '', '"\'')).strip() for s in decision_options.split(',')]
+                for decision in decision_options:
+                    email_message = note.content[f'{decision.lower().replace(" ", "_")}_email_content']
+                    conference.send_decision_notifications(decision, email_message)
```

Comment:
it seems for every decision you are loading all the papers and all the decisions. I think `send_decision_notifications` implementation can send emails to all the decisions options